src/Store/CompletionStoreConnector.re

```diff
@@ -6,106 +6,151 @@
 
 open EditorCoreTypes;
 open Oni_Core;
-open Oni_Core.Utility;
-module Model = Oni_Model;
+open Oni_Model;
+open Actions;
 
-module Actions = Model.Actions;
-module Animation = Model.Animation;
-module Quickmenu = Model.Quickmenu;
+module Option = Utility.Option;
+module VimEx = Utility.VimEx;
+module IndexEx = Utility.IndexEx;
 
-open Actions;
+module Log = (val Log.withNamespace("Oni2.CompletionStore"));
 
-type lastCompletionMeet = {
-  completionMeetBufferId: int,
-  completionMeetLine: Index.t,
-  completionMeetColumn: Index.t,
-};
+module Effects = {
```

Comment:
I like this style of putting effects in an `Effects` module 👍

Reply:
Doesn't work if the effects depend on something passed in through `start` though, unfortunately :confused: